test/end-to-end/pages/deleteBlueprint.js

```diff
@@ -25,17 +25,17 @@ module.exports = class deleteBlueprint extends MainPage {
 
     // ---- Page element selector ---- //
     // Page Title
-    this.labelPageTitle = `${this.headerElement} h4[class="modal-title"]`;
+    this.labelPageTitle = `${this.headerElement} [data-id="myModalLabel"]`;
 
     // X Close Button
-    this.btnXClose = `${this.headerElement} span[class="pficon pficon-close"]`;
+    this.btnXClose = `${this.headerElement} .pficon-close`;
 
     // Blueprint Name label
     this.labelBlueprintName = `${this.bodyElement} p strong`;
 
     // Delete and Cancel button
-    this.btnDelete = `${this.footerElement} button[class="btn btn-danger"]`;
-    this.btnCancel = `${this.footerElement} button[class="btn btn-default"]`;
+    this.btnDelete = `${this.footerElement} #btn-danger`;
+    this.btnCancel = `${this.footerElement} #btn-default`;
```

Comment:
Related to a previous comment, since we can probably rely on the classes in these elements to identify them, then we can use the class selectors here. For example:
```
    this.btnDelete = `${this.footerElement} .btn-danger`;
    this.btnCancel = `${this.footerElement} .btn-default`;
```